discovery-provider/alembic/versions/7b3f27e78b84_add_remixes_table.py

```diff
@@ -17,8 +17,8 @@
 
 
 def upgrade():
-    op.create_table('credits',
-        # A child track "credits" parent track (and can credit many)
+    op.create_table('remixes',
+        # A child track "remixes" parent track (and can remixes many)
         sa.Column('parent_track_id', sa.Integer(), nullable=False, index=True),
```

Comment:
@jowlee @raymondjacobson @piazz i don't think we should index these columns in the join tables just yet. for context, our tracks table has zero indexes at the moment, neither does reposts or saves which we join a bunch. wdyt?